histomicstk/features/ComputeFSDs.py

```diff
@@ -0,0 +1,253 @@
+import collections
+import numpy as np
+import pandas as pd
+from skimage.measure import regionprops
+from skimage.segmentation import find_boundaries
+
+
+def ComputeFSDs(im_label, K=128, Fs=6, Delta=8, rprops=None):
+    """
+    Calculates `Fourier shape descriptors` for each objects.
+
+    Parameters
+    ----------
+    im_label : array_like
+        A labeled mask image wherein intensity of a pixel is the ID of the
+        object it belongs to. Non-zero values are considered to be foreground
+        objects.
+    K : int, optional
+        Number of points for boundary resampling to calculate fourier
+        descriptors. Default value = 128.
+    Fs : int, optional
+        Number of frequency bins for calculating FSDs. Default value = 6.
+    Delta : int, optional
+        Used to dilate nuclei and define cytoplasm region. Default value = 8.
+
+    rprops : output of skimage.measure.regionprops, optional
+        rprops = skimage.measure.regionprops( im_label ). If rprops is not
+        passed then it will be computed inside which will increase the
+        computation time.
```

Comment:
@slee172 @cooperlab Optionally passing in output of regionprops to speed up computational time.